src/main/fc/runtime_config.c

```diff
@@ -53,7 +53,8 @@ const char *armingDisableFlagNames[]= {
     "MSP",
     "PARALYZE",
     "GPS",
-    "ARMSWITCH"
+    "RESC",
```

Comment:
You have 11 characters to play with here so maybe something more descriptive like "RESCUE SW" would be better.

Reply:
agreed!